crates/steel/src/ethereum.rs

```diff
@@ -63,10 +63,9 @@ pub static ETH_MAINNET_CHAIN_SPEC: LazyLock<EthChainSpec> = LazyLock::new(|| Cha
     ]),
 });
 
-// TODO: Make this a function of the spec ID.
-/// [ChainSpec] for an Anvil chain using the Cancun EVM.
-pub static ANVIL_CHAIN_SPEC: LazyLock<ChainSpec<SpecId>> =
-    LazyLock::new(|| ChainSpec::new_single(31337, SpecId::CANCUN));
+/// [ChainSpec] for a custom Steel Testnet using the Prague EVM.
+pub static TESTNET_CHAIN_SPEC: LazyLock<ChainSpec<SpecId>> =
```

Comment:
Should this be hidden behind a feature flag to prevent accidental use?

Reply:
I personally think not. It's clearly labeled and it wouldn't lead to immediate security failures if used. With the onchain config checks, it would also be caught failure early in any deployment process.